docker/build_scripts/install-entrypoint.sh

```diff
@@ -1,15 +1,16 @@
-#!/bin/bash
+#!/bin/sh
 # Install entrypoint:
-# make sure yum is configured correctly and linux32 is available on i686
+#   - make sure yum is configured correctly and linux32 is available on i686
+#   - install bash on Alpine as most scripts require it
 
 # Stop at any error, show all commands
-set -exuo pipefail
+set -exuo
```

Comment:
```suggestion
set -exu
```

The `-o` was for `-o pipefail`. That being said I don't know much about "raw" POSIX shell.

Reply:
Oops sorry, I missed that.